common/locales/en/pets.json

```diff
@@ -1,10 +1,12 @@
 {
     "pets": "Pets",
     "petsFound": "Pets Found",
+    "petsFoundEnding": "Pets Found",
     "rarePets": "Rare Pets",
     "questPets": "Quest Pets",
     "mounts": "Mounts",
     "mountsTamed": "Mounts Tamed",
+    "mountsTamedEnding": "Mounts Tamed",
```

Comment:
If this is important, I feel like we should just construct the entire string. This should also help for cases where the ordering of numbers is different than in English. 

String:

```json
"stableMountMasterProgress": "Mount Master Progress: <%= number => tamed"
```

Jade file:
```jade
h4 {{:: env.t('stableMountMasterProgress', { number: mountMasterProgress }) }}
````